src/app/services/views/ServiceTrayView.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <AppView>
+    <template #title>
+      <div class="tray-title-wrapper">
+        <img
+          aria-hidden="true"
+          src="@/assets/images/icon-wifi-tethering.svg?url"
+        >
```

Comment:
Is this icon going to be common to all tray views do you know?

Also given we are using `aria-hidden` would it be better to use CSS to add this icon?

Reply:
> Is this icon going to be common to all tray views do you know?

No. It just so happens that we currently have the same icon (in the main overview) for both services and data plane proxies. Ideally they’d be different, but that’s for another time.

> Also given we are using `aria-hidden` would it be better to use CSS to add this icon?

I find it easier to deal with them like this.